src/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/Exponential.cpp

```diff
@@ -1,16 +1,13 @@
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 /// @project        Open Space Toolkit ▸ Physics
-/// @file           OpenSpaceToolkit/Physics/Environment/Atmospheric/Exponential.cpp
+/// @file           OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/Exponential.cpp
```

Comment:
There's a similar situation in the Gravity modeling with the `Spherical` model. If we decide to move this one, I think we should move that one as well.